rsmtool/utils/notebook.py

```diff
@@ -30,14 +30,21 @@ def float_format_func(num, prec=3):
     prec: int, optional
         The number of decimal places to use when displaying the number.
         Defaults to 3.
+    scientific: bool, optional
+        Whether to display the number in scientific notiation if
```

Comment:
```suggestion
        Whether to display the number in scientific notation if
```

Reply:
@damien2012eng  can you fix this in your PR? I don't feel like running another build unless you also find some more typos? :)